mmv1/templates/terraform/examples/firebase_apple_app_basic.tf.erb

```diff
@@ -0,0 +1,22 @@
+resource "google_project" "default" {
+	provider = google-beta
+
+	project_id = "tf-test%{random_suffix}"
+	name       = "tf-test%{random_suffix}"
+	org_id     = "<%= ctx[:test_env_vars]['org_id'] %>"
+}
+
```

Comment:
we probably don't need to re-create the Project each time, if the App is being deleted (as per above comments)